hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieDefaultTimeline.java

```diff
@@ -100,6 +102,12 @@ public HoodieTimeline filterCompletedInstants() {
     return new HoodieDefaultTimeline(instants.stream().filter(HoodieInstant::isCompleted), details);
   }
 
+  @Override
+  public HoodieTimeline filterCompletedExcludeDeletePartitionInstants() {
```

Comment:
Making the API name more explicit helps explain the use case.

```suggestion
  public HoodieTimeline filterCompletedInstantsWithSchema() {
```